tilequeue/process.py

```diff
@@ -512,6 +522,34 @@ def _calculate_scale(scale, coord, nominal_zoom):
         return scale
 
 
+def _load_inline_layer(layer_path):
+    # we can optionally load geojson layers from file all we need is a file path. we expect the coordinates to already
+    # be in mercator projection and we only fill out the minimal structure of the feature tuple, ie there's no ID. if
+    # the path is given and the file exists, we expect to be able to load the layer's features. so if the json is
+    # malformed or shapely can't make sense of the geom then we'll raise
+    features = []
+
+    # has to exist and has to be geojson for now
+    if not os.path.isfile(layer_path) or not layer_path.endswith('.geojson'):
+        return features
+
+    # load the geojson into a shapely geometry collection
+    with open(layer_path) as fh:
+        fc = json.load(fh)
+        # skip if this isnt pseudo mercator
+        if fc['crs']['properties']['name'] != 'urn:ogc:def:crs:EPSG::3857':
+            return features
+        gc = GeometryCollection([shape(feature['geometry']) for feature in fc['features']])
+
+    # add the features geometries with their properties (values must be strings) in tuples
+    for geom, feat in itertools.izip(gc.geoms, fc['features']):
+        properties = {k: str(v) if type(v) is not bool else str(v).lower() for k, v in
```

Comment:
maybe use another name for `v` in the `for k,v` expression to increase readability?